index.html

```diff
@@ -1,54 +1,35 @@
 <html lang="en">
-  <head>
+<head>
     <title>Guess the Number</title>
-    <link
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      rel="stylesheet"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
-    />
-    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body class="text-center">
-    <nav class="navbar navbar-dark bg-dark">
-      <!-- Navigation bar content -->
-      <a class="navbar-brand" href="#">Guess The Number</a>
-    </nav>
-
-    <main>
-      <h1 class="banner p-3">1...100</h1>
-
-      <div class="form-group">
-        <input
-          id="number-guess"
-          class="form-control form-control-lg"
-          type="number"
-          placeholder="What's your guess?"
-        />
-      </div>
-
-      <div id="result">
-        <!-- This is where the result will be -->
-      </div>
-
-      <form>
-        <button
-          type="button"
-          id="number-submit"
-          class="btn btn-lg btn-secondary"
-        >
-          Check Me
-        </button>
-
-        <button type="button" id="restart-game" class="btn btn-lg btn-light">
-          Restart
-        </button>
-      </form>
-
-      <div id="history">
-        <!-- This is where the history will be -->
-      </div>
-    </main>
-  </body>
-</html>
+</head>
+    <body class="text-center">
+        <nav class="navbar navbar-dark bg-dark">
+            <!-- Navigation bar content -->
+            <a class = "navbar-brand" href="#">Guess The Number</a>
+        </nav>
+        <div class="header">
+            <h6 class="banner p-3">Try guessing a number from 1,2,3,...,100</h6>
+        </div>
```

Comment:
- [ ] Don't add two banners. Delete one. Have only one banner!

- [ ] Have the title of the banner - Guess the Number

If you like you can work on the font style and size of the banner to look dashing. Maybe UI Designers would use Photoshop to create their banners. They may also use canva.com. Overall design looks good.